.github/workflows/release.yml

```diff
@@ -78,6 +78,7 @@ jobs:
           args: release --clean -f ${{ vars.GORELEASER_CONFIG_PATH }} ${{ steps.prepare_goreleaser_extra_commands.outputs.EXTRA_GORELEASER_COMMANDS }}
         env:
           AUR_KEY: ${{ secrets.AUR_KEY }}
+          DISABLE_CHANGELOG: ${{ vars.DISABLE_CHANGELOG}}
```

Comment:
where is it declared?

Reply:
settings -> secrets and variables -> Actions -> variables